gitlab-ce/files/templates/gitlab-ssl.yml

```diff
@@ -67,7 +67,8 @@ objects:
             command:
             - bash
             - -c
-            - /opt/gitlab/embedded/bin/openssl req -newkey rsa:2048 -nodes -keyout /etc/gitlab/${APPLICATION_HOSTNAME}.key -x509 -days 365 -out /etc/gitlab/${APPLICATION_HOSTNAME}.crt -subj "/C=GB/ST=London/L=London/O=Container and PaaS CoP/CN=${APPLICATION_HOSTNAME}"
+            - test -e "/etc/gitlab/${APPLICATION_HOSTNAME}.crt" && { echo "Custom
```

Comment:
The space here is causing issue with the template

Reply:
Solved, thanks for reviewing @sabre1041